### PR TITLE
chore: remove unused things, fix some

### PR DESCRIPTION
**Pull request by wass3r**
mainly removes unused code. might be overzealous? 🤷🏼

### CODE REVIEW

File: src/elm/Api.elm [hunk 1]
```diff
@@ -447,26 +432,6 @@ updateCurrentUser model body =
...
-{-| getAllRepositories : used in conjuction with 'tryAll', it retrieves all pages of the resource
-
-    Note: the singular version of the type/decoder is needed in this case as it turns it into a list
-
--}
-getAllRepositories : PartialModel a -> Request Repository
-getAllRepositories model =
-    -- we using the max perPage setting of 100 to reduce the number of calls
-    get model.velaAPI (Endpoint.Repositories (Just 1) (Just 100)) decodeRepository
-        |> withAuth model.session
```
Comment (davidvader) [inline src/elm/Api.elm:467]: I know this is in Draft but we will be using `getAllRepositories` in the Overview refactor coming soon 👍

**Comment by davidvader** [comment]
I love it <3

File: src/elm/Api.elm (hunk 1 above)
Comment (wass3r) [reply src/elm/Api.elm]: Readded

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by davidvader — APPROVED** [review]

**Pull request merged**